test/hotspot/jtreg/compiler/c2/irTests/AndINodeIdealizationTests.java

```diff
@@ -38,22 +38,24 @@ public static void main(String[] args) {
         TestFramework.run();
     }
 
-    @Run(test = { "test1" })
+    @Run(test = { "test1", "test2" })
     public void runMethod() {
         int a = RunInfo.getRandom().nextInt();
+        int b = RunInfo.getRandom().nextInt();
 
         int min = Integer.MIN_VALUE;
         int max = Integer.MAX_VALUE;
 
-        assertResult(0);
-        assertResult(a);
-        assertResult(min);
-        assertResult(max);
+        assertResult(0, 0);
+        assertResult(a, a);
```

Comment:
```suggestion
        assertResult(a, b);
```
I assume you wanted this? Otherwise `b` is useless ;)